evap/student/templates/student_vote.html

```diff
@@ -173,7 +176,7 @@ <h3 class="mb-3">{{ evaluation.full_name }} ({{ evaluation.course.semester.name
 
         <script type="module">
             import {initTextAnswerWarnings} from "{% static 'js/text-answer-warnings.js' %}";
-
+            const lastSavedStorageKey = "student-vote-{{evaluation.id}}-{{request.user.id}}"
```

Comment:
since you added the evaluation id... should that also be added to sysiphus' customKeySuffix?

Reply:
for the vote form fields the evaluation id is already part of the key because sisyphus put the complete url into the key. or do you mean a different place?

Reply:
no, that sounds good.